ReleaseNotes.md

```diff
@@ -1,224 +1,348 @@
-* http://kb.mozillazine.org/Security.fileuri.strict_origin_policy
-* http://ejohn.org/blog/tightened-local-file-security/
+When running from your local machine, it is recommended to use a web server for accesing the demos. Several of the demos make use of AJAX calls; which typically are not allowed by
+the browser when run from the local file system.
 
 ## License ##
 
 Fluid Infusion is licensed under both the ECL 2.0 and new BSD licenses.
 
 More information is available in our [wiki](http://wiki.fluidproject.org/display/fluid/Fluid+Licensing).
-
-
 ## Third Party Software in Infusion ##
 
 This is a list of publicly available software that is redistributed with Fluid Infusion,
 categorized by license:
 
+### Apache 2.0 ###
+
+* [`fluid.load.scripts` is based on Jake Archibald's script loading example](http://www.html5rocks.com/en/tutorials/speed/script-loading/#toc-dom-rescue)
+* [Open Sans Light font](http://www.google.com/fonts/specimen/Open+Sans)
+
 ### MIT License ###
+
+* [Buzz v1.1.0](http://buzz.jaysalvat.com)
 * [Foundation v6.2.3](http://foundation.zurb.com/index.html)
 * [HTML5 Boilerplate v4.3](http://html5boilerplate.com/)
+* [html5shiv v3.7.2](https://code.google.com/p/html5shiv/)
 * [jQuery v3.1.0](http://jquery.com/)
-* [jQuery UI (Core; Interactions: draggable, resizable; Widgets: button, checkboxradio, controlgroup, dialog, mouse, slider, tabs, and tooltip) v1.12.1](http://ui.jquery.com/)
+* [jQuery Mockjax v2.2.1](https://github.com/jakerella/jquery-mockjax)
 * [jQuery QUnit v1.12.0](http://qunitjs.com)
 * [jQuery QUnit Composite v1.0.1](https://github.com/jquery/qunit-composite)
-* [jQuery Mockjax v2.2.1](https://github.com/jakerella/jquery-mockjax)
 * [jQuery scrollTo v1.4.2](http://flesler.blogspot.com/2007/10/jqueryscrollto.html)
 * [jQuery Touch Punch v0.2.2](http://touchpunch.furf.com/)
-* [jquery.simulate](https://github.com/eduardolundgren/jquery-simulate)
+* [jQuery UI (Core; Interactions: draggable, resizable; Widgets: button, checkboxradio, controlgroup, dialog, mouse, slider, tabs, and tooltip) v1.12.1](http://ui.jquery.com/)
 * [jquery.selectbox v0.5 (forked)](https://github.com/fluid-project/jquery.selectbox)
+* [jquery.simulate](https://github.com/eduardolundgren/jquery-simulate)
 * [Micro Clearfix](http://nicolasgallagher.com/micro-clearfix-hack/)
 * [Normalize v4.1.1](https://necolas.github.io/normalize.css/)
-* [Buzz v1.1.0](http://buzz.jaysalvat.com)
-* [html5shiv v3.7.2](https://code.google.com/p/html5shiv/)
-
-### zlib/libpng License ###
-* [fastXmlPull is based on XML for Script's Fast Pull Parser v3.1](http://wiki.fluidproject.org/display/fluid/Licensing+for+fastXmlPull.js)
-
-### ECL 2.0 ###
-* Sample markup and stylesheets from [Sakai v2.5](http://sakaiproject.org)
-
-### Apache 2.0 ###
-* [Open Sans Light font](http://www.google.com/fonts/specimen/Open+Sans)
 
 ### Public Domain ###
+
 * Douglas Crockford's [JSON.js (from 2007-11-06)](http://www.json.org/)
```

Comment:
As far as I can tell, we no long distribute Crockford's old JSON parser with Infusion, right?